### ISSUE TITLE
Feature request: Add columns at specific position

### ISSUE THREAD

**Issue by aik099** [TablePlus/TablePlus#981]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):

MySQL 5.x

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):

194

3. The steps to reproduce this issue:

When pressing `Command+I` shortcut (or `+ Column` button), then new column is added at the end of the table even when I've specifically have selected a particular column after which new column should be added.

**Comment by huyphams** [comment]
Thanks for the suggestion, this is one of the known issues. You can change the number in the # column as a temporary solution. We will support insert before/after and drag/drop later.

**Comment by aik099** [comment]
Thank you.

**Comment by huyphams** [comment]
Please download this patch, it will allow you to add a column at position: https://www.dropbox.com/s/kjeon87wr82g2ps/TablePlus.dmg?dl=0

**Comment by aik099** [comment]
Also related to column adding (my license doesn't cover the latest release, so I have no idea if it works like that already):

When you add several columns at once with positions specified, then it's handled not like an atomic operation, and because of that number I've specified as column position is reevaluated after each of the columns were added. Because of this columns are added not one after another, but with gaps. I can see that already if I look at to-be-executed sqls before Command+S is pressed.

Example:

Table structure before:
1 FieldA
2 FieldB
3 FieldC
4 FieldD

I want to have table structure like this:
1 FieldA
2 FieldB
3 FieldB_NEW1
4 FieldB_NEW2
5 FieldC
6 FieldD

And therefore I specify this column order for "_NEW*" columns:
1 FieldA
2 FieldB
3 FieldB_NEW1
4 FieldB_NEW2
3 FieldC
4 FieldD

At the end I've get this instead:

1 FieldA
2 FieldB
3 FieldB_NEW1
4 FieldC
5 FieldB_NEW2
6 FieldD

Thank you.
